base/stat.jl

```diff
@@ -61,13 +61,14 @@ show(io::IO, st::StatStruct) = print(io, "StatStruct(mode=$(oct(filemode(st),6))
 
 # stat & lstat functions
 
-const stat_buf = Array{UInt8}(ccall(:jl_sizeof_stat, Int32, ()))
+const stat_buf_sz = Int(ccall(:jl_sizeof_stat, Int32, ()))
```

Comment:
It does look correct. I'm just saying that we don't need to allocate an array here since the size is known at compile time.